### PR TITLE
Issue #1052

### PR DESCRIPTION
**Pull request by jcuanm**
I am able to get the user to redirect to the login page if they want to access the Bookmarklet.

### CODE REVIEW

**Comment by horak** [comment]
Just so I can understand what's going on here, do you mind explaining a little more of what this means? If someone wants to access the bookmarket, why would you redirect them to the home page? 

Is there a particular issue this fixes?

I feel like this could be very important so nice work, just curious what exactly is going on.

**Comment by jcuanm** [comment]
No problem! Right now, the Bookmarklet takes you to a page that tells you to log in before using the Bookmarklet. What I'm trying to do is redirect the user directly to the log in page so that they don't have to waste extra clicks. Once the user successfully logs in, it redirects them to the homepage.

File: web/js/lib/visualsearch/visualsearch.js [hunk 1]
```diff
@@ -156,6 +156,7 @@ VS.ui.SearchBox = Backbone.View.extend({
   // Used to launch a search. Hitting enter or clicking the search button.
   searchEvent : function(e) {
     var query = this.value();
+    console.log(query);
```
Comment (horak) [inline web/js/lib/visualsearch/visualsearch.js]: It's good practice to remove these console logs that you put in while debugging. 👍

**Comment by horak** [comment]
Nice work! Let me know when it is ready for testing.

**Comment by jcuanm** [comment]
This is ready for merging

**Pull request merged**